content/en/docs/tutorials/verify.md

```diff
@@ -0,0 +1,184 @@
+
+## Verify image signature with Notation CLI
+ [Verify the image signature as a software consumer](https://ratify.dev/blog/sign-and-verify-image-with-notation-ratify/#verify-the-image-signature-as-a-software-consumer) with the Notation CLI before using it.
+
+``` 
+notation verify $IMAGE
+```
+You can also check the signature digest and inspect the signature and its certificate information to make sure the image is produced by a trusted identity.
+
+```
+notation inspect $IMAGE
+```
+
+## Verifying a container image with Ratify 
+As a prerequisite, you’ll need:
+- Kubernetes v1.20 or higher (You can use [minikube](https://minikube.sigs.k8s.io/docs/start/) if you are new to Kubernetes)
+- Helm v3
+
+Using Ratify will ensure that only signed images are deployed to Kubernetes.
+Set up Gatekeeper with [external data.](https://open-policy-agent.github.io/gatekeeper/website/docs/externaldata)
+
+```console
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+
+helm install gatekeeper/gatekeeper  \
+   --name-template=gatekeeper \
+   --namespace gatekeeper-system --create-namespace \
+   --set enableExternalData=true \
+   --set validatingWebhookTimeoutSeconds=5 \
+   --set mutatingWebhookTimeoutSeconds=2
+```
+Install the lastest version of Ratify. Specify the certificate generated by Notation for verification purposes.
+
+```console
+helm repo add ratify https://deislabs.github.io/ratify
+
+helm install ratify \
+   ratify/ratify --atomic \
+   --namespace gatekeeper-system \
+   --set-file notaryCert=ghcr-networks.io.crt
+```
+Apply the constrait to enforce Gatekeeper policy to allow only signed images can be deployed on Kubernetes:
+
+```console
+kubectl apply -f https://deislabs.github.io/ratify/library/default/template.yaml
+kubectl apply -f https://deislabs.github.io/ratify/library/default/samples/constraint.yaml
+```
+Deploy the sample image signed by Notation. Ratify will verify if this image has a valid signature.
+
+```console
+kubectl run ratify-signed --image=$IMAGE
+/net-monitor@sha256:27c0290c485140c3c998e92c6ef23fba2bd9f09c8a1c7adb24a1d2d274ce3e8e
+pod/demo created
+```
+Check the signature associated with the image.
+
+```
+notation ls $IMAGE
+```
+Deploy an unsigned image to Kubernetes cluster. The deployment has been denied since the image has not been signed and doesn't meet the deployment criteria.
+
+```console
+$ kubectl run demo --image=ghcr.io/feynmanzhou/notation/alpine@sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870
```

Comment:
Change the image to something from a trusted, reputable repo, such as https://github.com/openzipkin/docker-alpine#readme

Reply:
This also affects subsequent steps and command output